docs: refine docs further (#6)
prisma/pulse-starter: README.md

# Prisma Pulse starter project

> This repository has been created to help you get started with [Pulse](https://prisma.io/pulse). You will be able to use this project with any Pulse-ready PostgreSQL database. This project comes with a basic [`prisma.schema`](./prisma/schema.prisma) file as well as a Pulse subscription found in the [`index.ts`](./index.ts) file.

Reviewer: ```suggestion
This repository has been created to help you get started with [Pulse](https://prisma.io/pulse). You will be able to use this project with any Pulse-ready PostgreSQL database. This project comes with a basic [`schema.prisma`](./prisma/schema.prisma) file as well as a Pulse subscription found in the [`index.ts`](./index.ts) file.
```

> ## Table of contents
> 
> - [Prerequisites](#prerequisites)
> - [Local development](#local-development)
>   - [1. Clone the respository](#1-clone-the-respository)
>   - [2. Create and fill out a `.env` file](#2-create-and-fill-out-a-env-file)
>   - [3. Run the database migration](#3-run-the-database-migration)
>   - [4. Start the Pulse subscription](#4-start-the-pulse-subscription)
>   - [5. Test the subscription](#5-test-the-subscription)
> - [Deployment](#deployment)
>   - [Deploy the Pulse Starter Project on railway.app](#deploy-the-pulse-starter-project-on-railwayapp)
> - [More example Pulse projects](#more-example-pulse-projects)
> - [More information about Pulse](#more-information-about-pulse)

Reviewer: I'd drop the TOC, the READMe isn't that long that it feels necessary.

```suggestion
```

## Prerequisites

To successfully run the project, you will need the following:

> - A Prisma Pulse-compatible database
>   - Refer to our comprehensive guide [here](https://www.prisma.io/docs/pulse/database-setup) for setup instructions, or deploy a Pulse-ready database on Railway [![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/pulse-pg?referralCode=VQ09uv)
> - An active [Prisma Data Platform](https://pris.ly/pdp) account for obtaining a Prisma Pulse API Key

Reviewer: ```suggestion
- The **connection string** of a Pulse-compatible database (if you don't have one yet, you can configure your database following the instructions in our [docs](https://www.prisma.io/docs/pulse/database-setup) or [use a Railway template](https://railway.app/template/pulse-pg?referralCode=VQ09uv))
- A **Pulse API key** which you can get by enabling a Pulse in a project in your [Prisma Data Platform](https://pris.ly/pdp) account
```

> ## Local development

Reviewer: ```suggestion
## Getting started
```

```ts
async function main() {
 const subscription = await prisma.user.subscribe();

 if (subscription instanceof Error) {
  throw subscription;
 }

 for await (const event of subscription) {
  console.log("just received an event:", event);
 }
}

> ```

Reviewer: indendation seems off here. let's use 2 spaces per indentation level, not one:

```ts
async function main() {
  const subscription = await prisma.user.subscribe();

  if (subscription instanceof Error) {
    throw subscription;
  }

  for await (const event of subscription) {
    console.log("just received an event:", event);
  }
}
```

```json
{
 "action": "create",
 "created": {
  "id": 1,
  "email": "test@prisma.io",
  "name": "test"
 }

> }

Reviewer: again, let's indent with two spaces per indentation level:

```json
{
  "action": "create",
  "created": {
    "id": 1,
    "email": "test@prisma.io",
    "name": "test"
  }
}
```

```

## Deployment

### Deploy the Pulse Starter Project on railway.app

> 

Reviewer: ```suggestion
You can also deploy this project on Railway by following the instructions in our [docs](https://www.prisma.io/docs/pulse/database-setup/railway#setup-with-template-prisma-pulse-db--app).
```

[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/pulse-starter?referralCode=VQ09uv)

> This project is used together with the [prisma/pulse-railway-pg-config](https://github.com/prisma/pulse-railway-pg-config) repo to build a template on [railway.app](https://railway.app).
> 
> [![Watch the setup template setup video](https://img.youtube.com/vi/0nt7CLDqYeY/0.jpg)](https://www.youtube.com/watch?v=0nt7CLDqYeY)

Reviewer: ```suggestion
```

>  For the most current configuration details, please refer to our [documentation](https://www.prisma.io/docs/pulse/database-setup/railway), as information in the video may be outdated.

## More example Pulse projects

- [Prisma Pulse examples](https://pris.ly/pulse-examples)

## More information about Pulse

- [Pulse documentation](https://pris.ly/pulse-docs)

> - [Pulse GA announcement blog post](https://pris.ly/gh/pulse-ga)

Reviewer: ```
## Resources

- [Prisma Pulse examples](https://pris.ly/pulse-examples)
- [Pulse documentation](https://pris.ly/pulse-docs)
- [Pulse GA announcement blog post](https://pris.ly/gh/pulse-ga)
